eq-author-api/schema/tests/questionnaire.test.js

```diff
@@ -208,6 +216,18 @@ describe("questionnaire", () => {
       );
       expect(oldestIndex > newestIndex).toEqual(true);
     });
+    it("should not list unaccessible private questionnaires", async () => {
+      const user = {
+        id: "123",
+      };
+      await buildContext({}, user);
```

Comment:
I don't think we actually need to create these questionnaires

Reply:
I fixed the test so it checks in the right place if the questionnaire is in the list. Also added a test to check if the `publicQuestionnaire` is in the list as it was hard to know if the check for `privateQuestionnaire` was actually working.